homeassistant/components/songpal/media_player.py

```diff
@@ -102,7 +100,7 @@ async def async_service_handler(service):
             key: value for key, value in service.data.items() if key != ATTR_ENTITY_ID
         }
 
-        for device in hass.data[PLATFORM].values():
+        for device in hass.data[DOMAIN].values():
```

Comment:
We can use the platform service registration helper and avoid storing the entities.

https://developers.home-assistant.io/docs/dev_101_services#entity-services